packages/expo-build-properties/src/__tests__/android-test.ts

```diff
@@ -66,3 +72,38 @@ describe(updateAndroidProguardRules, () => {
     `);
   });
 });
+
+describe(withAndroidFlipper, () => {
+  it('should do nothing by default or if set to enabled', async () => {
+    const expoConfig: ExpoConfig = {
+      name: 'withAndroidFlipper',
+      slug: 'withAndroidFlipper',
+    };
+
+    withAndroidFlipper(expoConfig, {});
+    expect((expoConfig as ExpoConfigWithMods)?.mods?.android).toBeUndefined();
+
+    withAndroidFlipper(expoConfig, {
+      android: {
+        flipper: true,
+      },
+    });
+    expect((expoConfig as ExpoConfigWithMods)?.mods?.android).toBeUndefined();
+  });
+
+  it('should update the flipper version if requested', async () => {
+    const expoConfig: ExpoConfig = {
+      name: 'withAndroidFlipper',
+      slug: 'withAndroidFlipper',
+    };
+    const pluginConfig = {
+      android: {
+        flipper: '0.999.0',
+      },
+    };
+    withAndroidFlipper(expoConfig, pluginConfig);
+    expect((expoConfig as ExpoConfigWithMods)?.mods?.android?.gradleProperties).toBeInstanceOf(
+      Function
```

Comment:
is it feasible to test the version would be `0.999.0`?

Reply:
Yeah. Test added in the style of our build properties tests